internal/digest/yesterday_test.go

```
@@ -0,0 +1,496 @@
+		t.Fatalf("read first: %v", err)
+	}
+
+	// Add later-day events — yesterday must not pick them up because the
+	// closed-day window's date is still 2026-05-12. Per Run's contract,
+	// events with timestamps outside the day still flow through the source
+	// (Run does not filter by timestamp), but yesterday's *Meta.Date* and
+	// the closed-day immutability contract say once it's written for a
+	// given closed day, re-running with the same window+events-of-that-day
+	// must be a no-op.
+	laterEvents := append([]ChangeEvent(nil), events...)
+	laterEvents = append(laterEvents, ChangeEvent{
```

golangci-lint (2.12.2)

[error] ineffectual assignment to laterEvents

(ineffassign)